usr.sbin/pkg/config.c

```diff
@@ -500,7 +500,11 @@ config_init(const char *requested_repo)
 		c[REPOS_DIR].list = malloc(sizeof(*c[REPOS_DIR].list));
 		STAILQ_INIT(c[REPOS_DIR].list);
 		cv = malloc(sizeof(struct config_value));
+#ifdef COMPAT_64BIT
+		cv->value = strdup("/etc/pkg64");
+#else
 		cv->value = strdup("/etc/pkg");
```

Comment:
Similarly to the local base, we should use `/etc/pkg64c` for pure-capability binaries.